src/deepsparse/loggers/helpers.py

```diff
@@ -217,7 +218,7 @@ def access_nested_value(
             # indexing
             operator = int(string_operator)
 
-        value = value.__getitem__(operator)
+        value = _warn_if_array_or_tensor(value.__getitem__(operator))
```

Comment:
should separate checking and assignment since we don't want the helper function to have an effect
```suggestion
        value = value.__getitem__(operator)
        _warn_if_array_or_tensor(value)
```